docs/docs/keymaps/behaviors/mod-morph.md

```diff
@@ -64,6 +70,37 @@ Example:
 mods = <(MOD_LGUI|MOD_LSFT|MOD_RGUI|MOD_RSFT)>;
 ```
 
+### Binding Parameters
+
+The `binding-params` property determines how the parameters passed to the mod-morph behavior are passed on to the behaviors listed in `bindings`.
+It is an array which always has two `BINDING_PARAM(arg1,arg2)` elements - the first corresponds to the first behavior listed in `bindings`, the second corresponding to the second. The number chosen for `argX` determines what the Xth parameter passed to the behavior will be:
+
+- `1`: The Xth parameter passed will be the first parameter that the mod-morph behavior received
+- `2`: The Xth parameter passed will be the second parameter that the mod-morph behavior received
+- `0`: The Xth parameter will be that which is written in the `bindings` array.
+
+Note that any parameters in `bindings` behaviors which are to be replaced should be set to `PLACEHOLDER`.
```

Comment:
It is worth noting here that if the behavior takes one/zero params, you should use `0` for `arg2`/`arg2` and `arg1`.